src/Features/LanguageServer/Protocol/Features/Options/CodeActionOptionsStorage.cs

```diff
@@ -46,5 +45,9 @@ internal static CodeActionOptionsProvider GetCodeActionOptionsProvider(this IGlo
             "UseConditionalExpressionOptions",
             "ConditionalExpressionWrappingLength", CodeActionOptions.DefaultConditionalExpressionWrappingLength,
             storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ConditionalExpressionWrappingLength"));
+
+        public static Option2<bool> EnableConvertToRecord = new(
+            "ConvertToRecord", "EnableConvertToRecord", defaultValue: false,
+            storageLocation: new FeatureFlagStorageLocation("Roslyn.EnableConvertToRecord"));
```

Comment:
How the end user can change the option?

Reply:
@Youssef1313 The plan was to feature flag this and it'd be present in previews for 17.5; right now this doesn't even work as there's a few other things that need fixing first.